velox/functions/prestosql/aggregates/tests/SetAggTest.cpp

```diff
@@ -155,6 +160,185 @@ TEST_F(SetAggTest, groupBy) {
       {expected});
 }
 
+TEST_F(SetAggTest, decimal) {
```

Comment:
This method has become too long. Split this test into 2 for shortDecimal and longDecimal.

Reply:
Split the test into 2 for shortDecimal and longDecimal.